chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -896,4 +899,4 @@ Zotero.Translate.ItemGetter.prototype = {
 		return obj;
 	}
 }
-Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
\ No newline at end of file
+Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
```

Comment:
My pleasure. Should I submit a similar change for 4.0, or is 4.0 in maintenance mode?